src/pybind/mgr/dashboard/controllers/orchestrator.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+from . import ApiController, Endpoint, ReadPermission
+from . import RESTController, Task
+from ..security import Scope
+from ..services.orchestrator import OrchClient
+
+
+def orchestrator_task(name, metadata, wait_for=2.0):
+    return Task("orchestrator/{}".format(name), metadata, wait_for)
+
+
+@ApiController('/orchestrator', Scope.ORCHESTRATOR)
+class Orchestrator(RESTController):
+
+    @Endpoint()
+    @ReadPermission
+    def status(self):
+        orch = OrchClient()
+        return {
+            'available': orch.available()
```

Comment:
The orchestrator also provides a status message. might make sense to add it here.

Reply:
fixed